Content.Client/Xenoarchaeology/Ui/AnalysisConsoleMenu.xaml.cs

```diff
@@ -15,169 +24,235 @@ namespace Content.Client.Xenoarchaeology.Ui;
 public sealed partial class AnalysisConsoleMenu : FancyWindow
 {
     [Dependency] private readonly IEntityManager _ent = default!;
+    [Dependency] private readonly IResourceCache _resCache = default!;
+    [Dependency] private readonly IPlayerManager _playerManager = default!;
     [Dependency] private readonly IGameTiming _timing = default!;
 
-    public event Action? OnServerSelectionButtonPressed;
-    public event Action? OnScanButtonPressed;
-    public event Action? OnPrintButtonPressed;
-    public event Action? OnExtractButtonPressed;
-    public event Action? OnUpBiasButtonPressed;
-    public event Action? OnDownBiasButtonPressed;
+    private readonly ArtifactAnalyzerSystem _artifactAnalyzer;
+    private readonly XenoArtifactSystem _xenoArtifact;
+    private readonly AudioSystem _audio;
 
-    // For rendering the progress bar, updated from BUI state
-    private TimeSpan? _startTime;
-    private TimeSpan? _totalTime;
-    private TimeSpan? _accumulatedRunTime;
+    private readonly Entity<AnalysisConsoleComponent> _owner;
+    private Entity<XenoArtifactNodeComponent>? _currentNode;
 
-    private bool _paused;
+    private readonly List<(string NodeId, int ExtractedPoints)> _nodeExtractionValues = new();
```

Comment:
Its really not pretty sight. I was considering to totally re-do it into some meaningful separate UI element but ended up with leaving it be, as it will be a lot of work and i don't have any real vision of how new UI element should behave (and this PR is fat as a whale as it is).
Renaming it is a good idea. also gonna add small comment so next reader would not be as puzzled with all this logic as we were...